tenable/io/v3/was/folders.py

```diff
@@ -0,0 +1,94 @@
+'''
+Folders
+========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`Web Application Scanning v3 folders <was-v2-folders>` API.
+
+Methods available on ``tio.v3.was.folders``:
+
+.. rst-class:: hide-signature
+.. autoclass:: FoldersAPI
+    :members:
+'''
+import uuid
+from typing import Dict, List
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+
+
+class FoldersAPI(ExploreBaseEndpoint):
+    _path = 'api/v3/was/folders'
```

Comment:
There is still discussion on what the right naming convention should be here. It does seem that we will be moving forward with this structure though.